src/plugins/table-of-contents/table-of-contents-display.js

```diff
@@ -1,10 +1,11 @@
 import React, { useEffect, useState } from 'react';
+import Markdown from '../../components/markdown.js';
 import { sectionDisplayProps } from '../../ui/default-prop-types.js';
 
 export default function TableOfContentsDisplay({ content }) {
   const [nodes, setNodes] = useState([]);
 
-  const { minLevel, maxLevel } = content;
+  const { minLevel, maxLevel, caption } = content;
```

Comment:
Could we maybe rather call this `title`? 😬 
We always call it so in the other plugins, except for for `music-xml-viewer` and `pdf-viewer`, where it actually does not act as a title but as an explanation text.

Reply:
So, as we agreed, I repurposed the field to be named `text` and to be a block markdown field making things much more flexible for the editor.